### PR TITLE
Added information for additional inclusion of paths to search classes in

### PR DESCRIPTION
**Pull request by DocFX**
Update index.rst to add an example of additional paths to search, like the Twig Bridge provided Tailwind 2 form themes.

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -181,6 +181,31 @@ This represents the Tailwind configuration file:
     symfonycasts_tailwind:
         config_file: 'tailwind.config.js'
 
+If you include any other files containing classes to be detected outside the 
+default directories, like the `form themes provided with the Twig Bridge component <https://github.com/symfony/symfony/blob/HEAD/src/Symfony/Bridge/Twig/Resources/views/Form/tailwind_2_layout.html.twig/>`_,
```
Comment (bocharsky-bw) [inline doc/index.rst:185]: I think we should link the docs instead, how about this link? 
```suggestion
default directories, like the ``symfony/twig-bridge``'s `Tailwind CSS form theme <https://symfony.com/doc/current/form/tailwindcss.html>`_
```

File: doc/index.rst [hunk 2]
```diff
@@ -181,6 +181,31 @@ This represents the Tailwind configuration file:
     symfonycasts_tailwind:
         config_file: 'tailwind.config.js'
 
+If you include any other files containing classes to be detected outside the 
+default directories, like the `form themes provided with the Twig Bridge component <https://github.com/symfony/symfony/blob/HEAD/src/Symfony/Bridge/Twig/Resources/views/Form/tailwind_2_layout.html.twig/>`_, 
+untouched from the ``vendor`` directory, for instance, after changing your Twig configuration like this:
```
Comment (bocharsky-bw) [inline doc/index.rst:186]:
```suggestion
 from the untouched ``vendor/`` directory, after changing your Twig configuration like this:
```

File: doc/index.rst [hunk 3]
```diff
@@ -181,6 +181,31 @@ This represents the Tailwind configuration file:
...
+
+.. code-block:: yaml
+
+    # config/packages/twig.yaml
+    twig:    
+        form_themes:
+            - 'tailwind_2_layout.html.twig'
+
+...you can add them to your ``tailwind.config.js`` file, like this:
```
Comment (bocharsky-bw) [inline doc/index.rst:195]:
```suggestion
You can add them to your ``tailwind.config.js`` file like this:
```

**Review by bocharsky-bw — COMMENTED** [review]
Nice idea, thanks for improving the docs!

I left some comments, DYT?

File: doc/index.rst (hunk 1 above)
Comment (DocFX) [reply doc/index.rst]: That's better indeed, didn't know there was a dedicated page for this, or didn't find it!

File: doc/index.rst (hunk 2 above)
Comment (DocFX) [reply doc/index.rst:186]: My English might be deficient here: I thought the first one means "didn't copy/paste it in your project from vendor".

Second one seems to mean "not altering the vendor directory", which is by definition.

File: doc/index.rst (hunk 3 above)
Comment (DocFX) [reply doc/index.rst:195]: It's supposed to be a continuation from the unfinished sentence started above. Maybe that use of punctuation is not English. Is it?

Comment (bocharsky-bw) [reply doc/index.rst]: yeah, I see... I think with this simple way it will be better anyway

**Comment by bocharsky-bw** [comment]
Thank you for improving the docs!

**Pull request merged**